dbt/adapters/databricks/connections.py

```diff
@@ -726,6 +743,12 @@ def exponential_backoff(attempt: int) -> int:
             retry_timeout=(timeout if timeout is not None else exponential_backoff),
         )
 
+    @classmethod
+    def get_response(cls, cursor: DatabricksSQLCursorWrapper) -> DatabricksAdapterResponse:
```

Comment:
Where is this used?  I don't see it in this PR.

Reply:
`get_response` is called by the `add_query` and `execute` methods in this same file. Previously we inherited the `get_response` behaviour of databricks-spark (which we are still a fork of...for now).

Reply:
Ack.  Github PRs plus python not using an override keyword makes such uses hard to spot, thanks :).